lib/utils/url_launcher.dart

```diff
@@ -1,9 +1,15 @@
+import 'package:url_launcher/url_launcher.dart';
 import 'package:url_launcher/url_launcher_string.dart';
 
-void launchURL(String url) async {
-  if (await canLaunchUrlString(url)) {
-    await launchUrlString(url);
+void launchURL(String url, bool value) async {
+  if (url.startsWith('mailto')) {
+    final url = Uri.parse('mailto:support@circuitverse.org');
+    await launchUrl(url);
```

Comment:
For mail consider to open in external application.

Reply:
yes it opens in default mail app of smartphone

Reply:
i have sent you a video regarding all links on slack, please watch it , it will clear all doubts

Reply:
Ok then it's fine

Reply:
> @sainideepanshu199 remove the pubspec.lock file

shall i delete the file or revert the changes

Reply:
just delete the file and commit. It  will work.

Reply:
done